src/Components/Facility/HospitalList.tsx

```diff
@@ -343,7 +347,23 @@ export const HospitalList = (props: any) => {
                 )}
               </div>
               <div className="h-full w-full grow">
-                <div className="h-full flex flex-col justify-between w-full">
+                {isSmallScreen ? (
```

Comment:
```suggestion
                <div className="md:hidden">
```

We can use `md:hidden` here, to instruct CSS to hide this div for screen sizes >= 768px. Achieving the same result.